docs/advisors/checks/innodb_redo_logs_not_sized_correctly.md

```diff
@@ -0,0 +1,15 @@
+# InnoDB redo log size advisor
+## Description
+The configuration variable innodb_log_file_size is one of the most important settings with respect to innodb configuration.
```

Comment:
The innodb_log_file_size is for all of the log files.
The innodb_redo_log_capacity defines the size of the redo logs (measured in bytes).
https://dev.mysql.com/doc/refman/8.0/en/innodb-parameters.html#sysvar_innodb_redo_log_capacity

Reply:
The undo and redo logs have had several iterations through 5.7 and 8.0. I'm looking at the view from the latest version of 8.0. 
5.7 uses Innodb_log_file_size.

Reply:
8.0.30 introduced innodb_redo_log_capacitiy
https://www.percona.com/blog/how-to-calculate-a-good-mysql-redo-log-size-in-mysql-8/